app/editor/src/features/admin/filters/FilterList.tsx

```diff
@@ -10,21 +11,37 @@ import * as styled from './styled';
 export const FilterList: React.FC = () => {
   const navigate = useNavigate();
   const [{ initialized, filters }, { findAllFilters }] = useFilters();
+  const [{ filterFilter }] = useAdminStore();
 
   const [items, setItems] = React.useState<IFilterModel[]>(filters);
 
   React.useEffect(() => {
     if (!initialized) {
-      findAllFilters()
-        .then((data) => {
-          setItems(data);
-        })
-        .catch(() => {});
+      findAllFilters().catch(() => {});
     }
     // The api will cause a double render because findAllFilters(...) updates the store.
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [initialized]);
 
+  React.useEffect(() => {
```

Comment:
Remember filter so that returning to page is better.

Reply:
HAVE ALL MY APPROVAL!!!